packages/datagateway-download/src/downloadCart/urlBuilders/buildDatafileUrl.test.ts

```diff
@@ -0,0 +1,67 @@
+import axios from 'axios';
+import {
+  mockDatafiles,
+  mockedSettings,
+  mockFacilityCycles,
+} from '../../testData';
+import buildDatafileUrl from './buildDatafileUrl';
+
+describe('buildDatafileUrl', () => {
+  beforeEach(() => {
+    axios.get = jest.fn().mockResolvedValue({
+      data: [mockDatafiles[0]],
+    });
+  });
+
+  it('should return a generic URL to the parent dataset of the datafile', async () => {
+    const url = await buildDatafileUrl({
+      apiUrl: mockedSettings.apiUrl,
+      facilityName: mockedSettings.facilityName,
+      datafileId: 70,
+      facilityCycles: mockFacilityCycles,
+    });
+
+    expect(url).toBe('/browse/investigation/58/dataset/856/datafile');
+  });
+
+  it('should return an ISIS URL to the parent dataset of the datafile', async () => {
+    const url = await buildDatafileUrl({
+      apiUrl: mockedSettings.apiUrl,
+      facilityName: 'ISIS',
+      datafileId: 70,
+      facilityCycles: mockFacilityCycles,
```

Comment:
We don't need to pass this in in the tests anymore - leaving it here is potentially confusing